packages/system/ingress-nginx/Makefile

```diff
@@ -1,21 +1,14 @@
 NAME=ingress-nginx
-NAMESPACE=cozy-ingress-nginx
+NAMESPACE=cozy-$(NAME)
 
-show:
-	helm template --dry-run=server -n $(NAMESPACE) $(NAME) .
-
-apply:
-	helm upgrade -i -n $(NAMESPACE) $(NAME) .
-
-diff:
-	helm diff upgrade --allow-unreleased --normalize-manifests -n $(NAMESPACE) $(NAME) .
+include ../../hack/package-system.mk
 
 update:
 	rm -rf charts
-	helm repo add ingress-nginx https://kubernetes.github.io/ingress-nginx
-	helm repo update ingress-nginx
-	helm pull ingress-nginx/ingress-nginx --untar --untardir charts
+	helm repo add $(NAME) https://kubernetes.github.io/$(NAME)
+	helm repo update $(NAME)
+	helm pull $(NAME)/$(NAME) --untar --untardir charts
 	patch -p 3 < patches/add-metrics2.patch
-	rm -f charts/ingress-nginx/templates/controller-deployment.yaml.orig
-	rm -rf charts/ingress-nginx/changelog/
-	#sed -i '/  type:/a \  allocateLoadBalancerNodePorts: false' charts/ingress-nginx/templates/controller-service.yaml
+	rm -f charts/$(NAME)/templates/controller-deployment.yaml.orig
+	rm -rf charts/$(NAME)/changelog/
+	#sed -i '/  type:/a \  allocateLoadBalancerNodePorts: false' charts/$(NAME)/templates/controller-service.yaml
```

Comment:
I would rather keep update target without using variables, because commands here targets to concrete repositories, they are node depended on release name or namespace.